src/test/java/uk/gov/hmcts/reform/professionalapi/controller/internal/OrganisationInternalControllerTest.java

```diff
@@ -572,4 +576,21 @@ void testRetrieveOrgByPbaStatus() {
                 .getOrganisationsByPbaStatus(pbaStatus.toString());
     }
 
+    @Test
+    void testDeleteProfessionalUserFromOrganisation() {
+        DeleteUserResponse deleteUserResponse =
+            new DeleteUserResponse(STATUS_CODE_204, "The organisation has deleted successfully");
```

Comment:
wrong message you are trying delete user : The organisation has deleted successfully